contrib/build-certs.py

```
@@ -0,0 +1,165 @@
+    # expire in 7 days to avoid people using these in production
+    dt_activation = (datetime.utcnow() - timedelta(days=1)).isoformat()
+    dt_expiration = (datetime.utcnow() + timedelta(days=7)).isoformat()
+
+    # certificate authority
+    ca = "ACME"
+    ca_privkey = f"{ca}-CA.key"
+    ca_certificate = f"{ca}-CA.pem"
+    if not os.path.exists(ca_privkey):
+        print("generating private key...")
+        argv = ["certtool", "--generate-privkey", "--outfile", ca_privkey]
+        rc = subprocess.run(argv)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)